docs/reference/src/advanced/debugging_techniques.md

```diff
@@ -3,6 +3,10 @@
 
 On this page we share different techniques and tools we've built into Slint that help you track down different issues you may be running into, during the design and development.
 
+## Print Debugging Properties
+
+The `debug(...)` function accepts one or more properties and prints them to stdout, returning nothing. This function can be used in bindings.
```

Comment:
```suggestion
Use the [`debug()`](../language/builtins/functions.md#debug) function to print the value of properties to stderr.
```

I think the rest, i.e. that the function returns nothing, that it routes through the Platform trait's debug_log function for Rust, as well as an example of a `TouchArea`'s `clicked` handler that uses debug to print something, belongs into the documentation of the function.